src/request.rs

```diff
@@ -210,6 +210,49 @@ impl RequestBuilder {
         self
     }
 
+    /// Modify the query string of the URL.
+    ///
+    /// Modifies the URL of this request, adding the parameters provided.
+    /// This method appends and does not overwrite. This means that it can
+    /// be called multiple times and that existing query parameters are not
+    /// overwritten if the same key is used. The key will simply show up
+    /// twice. `.query([("foo", "a"), ("foo", "b")])` -> `"foo=a&boo=b"`
+    ///
+    /// ```rust
+    /// # use reqwest::Error;
+    /// # use std::collections::HashMap;
+    /// #
+    /// # fn run() -> Result<(), Error> {
+    /// let client = reqwest::Client::new();
+    /// let res = client.get("http://httpbin.org")
+    ///     .query([("lang", "rust")])
+    ///     .send()?;
+    /// # Ok(())
+    /// # }
+    /// ```
+    ///
+    /// # Warning
```

Comment:
Perhaps call this `Note` instead of `Warning`. It's not that something will go terribly wrong, it's just something to maybe read.